doc/classes/RenderingDevice.xml

```diff
@@ -460,6 +460,24 @@
 				This method does nothing.
 			</description>
 		</method>
+		<method name="get_driver_and_device_memory_report" qualifiers="const">
+			<return type="String" />
+			<description>
+				Returns string report in CSV format using the following methods:
+				- [method get_tracked_object_name]
+				- [method get_tracked_object_type_count]
+				- [method get_driver_total_memory]
+				- [method get_driver_allocation_count]
+				- [method get_driver_memory_by_object_type]
+				- [method get_driver_allocs_by_object_type]
+				- [method get_device_total_memory]
+				- [method get_device_allocation_count]
+				- [method get_device_memory_by_object_type]
+				- [method get_device_allocs_by_object_type]
+
+				This is only used by Vulkan in Debug builds and Godot must be started with --extra_gpu_memory_tracking.
```

Comment:
Avoid double paragraph break:

```suggestion
				Returns string report in CSV format using the following methods:
				- [method get_tracked_object_name]
				- [method get_tracked_object_type_count]
				- [method get_driver_total_memory]
				- [method get_driver_allocation_count]
				- [method get_driver_memory_by_object_type]
				- [method get_driver_allocs_by_object_type]
				- [method get_device_total_memory]
				- [method get_device_allocation_count]
				- [method get_device_memory_by_object_type]
				- [method get_device_allocs_by_object_type]
				This is only used by Vulkan in debug builds and Godot must be started with [code]--extra_gpu_memory_tracking[/code] [url=$DOCS_URL/tutorials/editor/command_line_tutorial.html]command line argument[/url].
```

Reply:
Given that `String::num_real` is not locale independent, I'm trying to avoid locales where "1,05" means "1.05".
No locale uses ";" inside numbers and the use of semicolon as separator is well supported by lots of software.

It is not really intended to be consumed by Godot again, although that should be possible.

The main intention of `get_driver_and_device_memory_report()` is to get a readable report that can be printed on a console and exported somewhere else, like a spreadsheet or a bug report.

There is nothing in `get_driver_and_device_memory_report()` that cannot be implemented by hand in gdscript using the methods in the XML documentation (`get_tracked_object_name`, `get_tracked_object_type_count`, etc); and using those functions that directly should be preferred for doing anything more complex (like presenting a table of memory consumption in Godot's UI).